inc/class-s3-uploads.php

```diff
@@ -67,6 +67,10 @@ public function tear_down() {
 	 */
 	public function register_stream_wrapper() {
 		if ( defined( 'S3_UPLOADS_USE_LOCAL' ) && S3_UPLOADS_USE_LOCAL ) {
+			if ( in_array( 's3', stream_get_wrappers() ) ) {
++                		stream_wrapper_unregister( 's3' );
++          		}
```

Comment:
I think these pluses are an artifact of a copy/paste from a patch and are the likely cause of tests failing.